build.fsx

```diff
@@ -127,6 +130,60 @@ Target "MultiNodeTests" (fun _ ->
 )
 
 //--------------------------------------------------------------------------------
+// NBench targets 
+//--------------------------------------------------------------------------------
+
+// Filter out assemblies which can't run on Linux, Mono, .NET Core, etc...
+let filterPlatformSpecificAssemblies (assembly:string) =
```

Comment:
@alexvaluyskiy on second thought, you're right it's not necessary because we don't have a CI task to even try to run them.  I'll take it out for now, but it might be helpful once we don't need to cherry pick inside of the `RunTests` target - especially if we need to filter inside other targets.